packages/typography/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "@kaizen/typography",
+  "version": "1.0.0",
+  "description": "Collection of Kaizen typography components",
+  "scripts": {
+    "prepublish": "tsc --project tsconfig.dist.json",
+    "build": "yarn clean && yarn prepublish",
+    "build:watch": "yarn clean && yarn prepublish --watch",
+    "clean": "rimraf '**/*.d.ts' '**/*.js' '**/*.map'"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cultureamp/kaizen-design-system.git",
+    "directory": "packages/typography"
+  },
+  "bugs": {
+    "url": "https://github.com/cultureamp/kaizen-design-system/issues"
+  },
+  "files": [
+    "**/*",
+    "!**/*.ts",
+    "!**/*.tsx",
+    "**/*.d.ts",
+    "!**/*.spec.*",
+    "!**/*.snap",
+    "!draft-templates",
+    "!stories",
+    "!docs",
+    "!tsconfig.dist.json"
+  ],
+  "author": "",
+  "private": false,
+  "license": "MIT",
+  "dependencies": {
+    "@kaizen/component-library": "^12.6.0",
```

Comment:
Seems like this is only used by the stylesheets just like `@kaizen/design-tokens`, so perhaps it can be moved into `peerDeps`? It would probably mean that `~kaizen/deprecated-component-library-helpers` should be added too though.

Reply:
I'm gonna have a quick go at removing any usages of component-library-helpers to see if I can remove the dependency on this completely.

Reply:
Ok that seems to have worked well. I just copied over the `fonts.scss` file, which makes sense to live here. Not sure if it should be in a folder 🤔

The only `component-library` reference left is a Box import in one of the stories files. Doesn't seem to care that it's not listed in the deps, but wonder if I should add to devDeps?

Reply:
Hmm... I wonder how it works to begin with. Does lerna actually create internal aliases for the packages in the repo? Of which case, then the dep probably doesn't need to exist at all. Sounds kinda funny to put it in a devDep though, since it actually is "needed" per se to build storybook. Maybe trying to build storybook might give us our answer?

Reply:
Storybook builds fine, so maybe we just leave it out 👍